scripts/mpinfoin.pl

```diff
@@ -53,26 +54,18 @@
 my $personinfohash;
 my $consinfohash;
 
-# Find latest register of members interests file
+# Change to raw data directory
 chdir mySociety::Config::get('RAWDATA');
```

Comment:
I think this can be removed now as well? It was only doing this to find the regmem files.